### PR TITLE
Login required fields

### PR DESCRIPTION
**Pull request by pgiraud**

### CODE REVIEW

File: powa/static/js/components/LoginView.vue [hunk 1]
```diff
@@ -7,14 +7,24 @@
         </v-toolbar>
         <v-form method="post" action="">
           <v-card-text>
-            <v-text-field name="user" label="Username"></v-text-field>
+            <v-text-field
+              name="user"
+              label="Username"
+              :rules="[rules.required]"
```
Comment (rjuju) [inline powa/static/js/components/LoginView.vue:13]: the username and password are actually not required as you can specify them on a per-server basis in the config file.

**Review by rjuju — CHANGES_REQUESTED** [review]

**Review by rjuju — APPROVED** [review]
Nice change, thanks!

Can you merge all the changes in a single commit too?

**Comment by pgiraud** [comment]
> Can you merge all the changes in a single commit too?

Done!

**Comment by rjuju** [comment]
thanks!

**Review by rjuju — APPROVED** [review]

**Pull request merged**